src/apis/disableReplyForAnnotations.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Disable reply for annotations determined by the function passed in as parameter
+ * @param Function that takes an annotation and returns if the reply of this annotations should be disabled.
```

Comment:
I think this should be @param {function}, but can't find jsDoc documentation about this :(